apps/condo/domains/common/utils/serverSchema/GqlToKnexBaseAdapter.js

```diff
@@ -0,0 +1,77 @@
+const has = require('lodash/has')
+
+class GqlToKnexBaseAdapter {
+    domainName = null
+    where = []
+    dayGroups = ['day', 'month', 'week', 'quarter', 'year']
+    dayGroup = 'day'
+    dateRange = { from: null, to: null }
+    groups = []
+    result = null
+
+    /**
+     * Formatting GQL expressions to Knex data structure
+     * Split up where conditions by structure described at https://knexjs.org/#Builder-where
```

Comment:
I think, we should document it more precisely:

```suggestion
     * Converts Keystone GraphQL `…WhereInput` conditions to shape, that will be used by working with Knex methods, described at https://knexjs.org/#Builder-where
```

Reply:
updated comment according to your suggestion